src/pyj/read_book/cfi.pyj

```diff
@@ -677,17 +661,18 @@ def scroll_to(cfi, callback, doc):  # {{{
 
             x = (point_.a*rect.left + (1-point_.a)*rect.right)
             y = (rect.top + rect.bottom)/2
-            x, y = viewport_to_document(x, y, ndoc)
+            x, y = scroll_viewport.viewport_to_document(x, y, ndoc)
```

Comment:
Everywhere this function is called, use ScrollViewport, for the reason described in the comment on cfi.pyj.